README.md

```diff
@@ -10,52 +10,94 @@
-By default, this package looks for your QuickBooks data in the `quickbooks` schema of your [target database](https://docs.getdbt.com/docs/running-a-dbt-project/using-the-command-line-interface/configure-your-profile). If this is not where your QuickBooks data is, add the following configuration to your `dbt_project.yml` file:
+## Step 3: Define database and schema variables
+By default, this package runs using your destination and the `quickbooks` schema of your [target database](https://docs.getdbt.com/docs/running-a-dbt-project/using-the-command-line-interface/configure-your-profile). If this is not where your QuickBooks data is (for example, if your QuickBooks schema is named `quickbooks_fivetran`), add the following configuration to your root `dbt_project.yml` file:
 
 ```yml
-# dbt_project.yml
+vars:
+    quickbooks_database: your_destination_name
+    quickbooks_schema: your_schema_name 
+```
 
-...
-config-version: 2
+## Step 4: Enabling/Disabling Models
+<details><summary>Expand for details</summary>
+Your QuickBooks connector might not sync every table that this package expects. This package takes into consideration that not every QuickBooks account utilizes the same transactional tables.
 
+By default, most variables' values are assumed to be `true` (with exception of  `using_purchase_order` and `using_credit_card_payment_txn`). In other to enable or disable the relevant functionality in the package, you will need to add the relevant variables:
+
+```yml
 vars:
-    quickbooks_database: your_database_name
-    quickbooks_schema: your_schema_name
+  quickbooks_source:
+    using_address: false # disable if you don't have addresses in QuickBooks
+    using_bill: false # disable if you don't have bills or bill payments in QuickBooks
+    using_credit_memo: false # disable if you don't have credit memos in QuickBooks
+    using_department: false # disable if you don't have departments in QuickBooks
+    using_deposit: false # disable if you don't have deposits in QuickBooks
+    using_estimate: false # disable if you don't have estimates in QuickBooks
+    using_invoice: false # disable if you don't have estimates in QuickBooks
+    using_invoice_bundle: false # disable if you don't have estimates in QuickBooks
+    using_journal_entry: false # disable if you don't have estimates in QuickBooks
+    using_payment: false # disable if you don't have estimates in QuickBooks
+    using_refund_receipt: false # disable if you don't have estimates in QuickBooks
+    using_transfer: false # disable if you don't have estimates in QuickBooks
+    using_vendor_credit: false # disable if you don't have estimates in QuickBooks
+    using_sales_receipt: false # disable if you don't have estimates in QuickBooks
+    using_purchase_order: true # enable if you want to include purchase orders in your staging models
+    using_credit_card_payment_txn: true # enable if you want to include credit card payment transactions in your staging models
 ```
+</details>
```

Comment:
These are a required step and we should probably not hide them in the collapsable section. Let's keep this step out of a collapse.